string target_name
---
int64 object_num
string[] object_list
